bindings/python/.evergreen/integ-setup.sh

```diff
@@ -28,7 +28,7 @@ MONGOCRYPT_DIR: "$MONGOCRYPT_DIR"
 EOT
 
 # Set up drivers-tools with a .env file.
-git clone https://github.com/mongodb-labs/drivers-evergreen-tools.git ${DRIVERS_TOOLS}
+git clone --branch DRIVERS-3032-fix-crypt_shared_loc https://github.com/blink1073/drivers-evergreen-tools.git ${DRIVERS_TOOLS}
```

Comment:
Once the PR for drivers-evergreen-tools is merged, this will have to be reverted.